x/gamm/keeper/multihop.go

```diff
@@ -1,365 +0,0 @@
-package keeper
```

Comment:
Note: diffed this whole file against `x/swaprouter/swap.go`. The few changes are summarized in `x/swaprouter/swap.go` per-method.